plugins/optimization-detective/class-od-link-collection.php

```diff
@@ -25,7 +25,7 @@
  *                   href?: non-empty-string,
  *                   imagesrcset?: non-empty-string,
  *                   imagesizes?: non-empty-string,
- *                   crossorigin?: ''|'anonymous'|'use-credentials',
+ *                   crossorigin?: 'anonymous'|'use-credentials',
```

Comment:
While the HTML spec allows for links the `crossorigin` attribute to be (cf. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)):

- `crossorigin` (boolean attribute, same as `anonymous`)
- `crossorigin=""` (empty value, same as anonymous`)
- `crossorigin=sooooo-secret` (invalid value, same as `anonymous`)
- `crossorigin=anonymous`
- `crossorigin=use-credentials`

This PR normalizes the first three cases to only use the fourth case, which aligns with the reflected DOM `crossOrigin` property. This will ensure that `OD_Link_Collection::OD_Link_Collection()` will be able to properly merge links for all `anonymous` links.